build.py

```diff
@@ -100,7 +103,11 @@ def initialize(project):
 
     project.set_property("copy_resources_target", "$dir_dist")
     project.get_property("copy_resources_glob").append("LICENSE")
-    project.get_property("filter_resources_glob").append("**/pybuilder/__init__.py")
+    project.get_property("filter_resources_glob").extend(["**/pybuilder/__init__.py",
+                                                          '**/pybuilder/plugins/python/sphinx_plugin.py'])
```

Comment:
We should get rid of this because this means that we hardcode the pybuilder project metadata in the plugin when building.
This works for pybuilder but not for other projects. If we ship like this, every other project using this plugin will have the replaced tags in the plugin file.

Would using properties not be sufficient? `sphinx_plugin_doc_author` etc.